src/libraries/EventsLib.sol

```diff
@@ -29,7 +29,7 @@ library EventsLib {
     event RootRevoked();
 
     /// @notice Emitted when rewards are claimed.
-    /// @param account The address for which rewards are claimd rewards for.
+    /// @param account The address for which rewards are claimed.
```

Comment:
```suggestion
    /// @param account The address on behalf of which rewards are claimed.
```

Reply:
The rewards are necessarily transferred to this account so Idk if `on behalf` works well here

Reply:
It still makes sense to me but it is certainly non blocking